src/services/jwt.service.js

```diff
@@ -0,0 +1,38 @@
+const jwt = require('jsonwebtoken');
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '5s',
```

Comment:
The access token expiration time is set to '5s', which is very short. Consider increasing the expiration time to a more practical duration, such as '1h' or '24h', depending on your application's requirements.